### ISSUE TITLE
css:hover not working on Android

### ISSUE THREAD

**Issue by ropoman** [timmywil/jquery.panzoom#170]
Hi,

I am panzooming this:
`<div class="wrapper">
  <img class="map" src=".." />
  <div class="items"><div class="item"> ...`
The items are positioned on the image by css left: and top: 
Using css:hover the items are opened and display more information. All works fine on the iPad, but the hover does not work on any Android device I have tried. Without panzoom the hover works on Android

Please help ;)

Thanks,
Ropoman

**Comment by ropoman** [comment]
I found out it works in Android's default browser, but not in Chrome.
Can't get this fiddle to run properly on tablet: http://jsfiddle.net/tnu5515x/4/

**Comment by ropoman** [comment]
Also does not work in Firefox

**Comment by TimmothyClaudius** [comment]
How are you hovering with a touch device? In general, you cannot hover on a touch device. Maybe if we know what you are trying to do, we can get a better idea of how to help.

**Comment by ropoman** [comment]
The hover is for desktop. On mobile devices the :hover is executed on touch. Just try the fiddle

**Comment by TimmothyClaudius** [comment]
I can't duplicate the issue on Chrome Mobile or Desktop editions, the fiddle appears to be working as intended. Maybe try updating Chrome?

Can anyone else replicate OP's issue?

**Comment by ropoman** [comment]
Which Android version have you tried?

**Comment by TimmothyClaudius** [comment]
I am using Chrome 43.0.2357.92

**Comment by ropoman** [comment]
Let me rephrase this: on which version of Android have you tried it?

**Comment by TimmothyClaudius** [comment]
Android 4.4.4. Either way, it sounds more like an issue outside of panzoom
itself.

**Comment by TimmothyClaudius** [comment]
Similar results on 4.1 and 4.3. I've not got a 5.x device however
